src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/integration/PoPUserOTPIntegrationTest.kt

```diff
@@ -269,19 +279,55 @@ class PoPUserOTPIntegrationTest : IntegrationTestBase() {
     every {
       SecureRandom.getInstanceStrong().nextLong(999999)
     } returns 123456
+    val nomsId = "G4161UF"
 
+    prisonerSearchApiMockServer.stubGetPrisonerDetails(nomsId, 200)
     popUserApiMockServer.stubPostPoPUserVerification(200)
+
+    webTestClient.post()
+      .uri("/resettlement-passport/popUser/onelogin/verify")
+      .bodyValue(
+        OneLoginData(
```

Comment:
is it possible to pass a json string instead of OneLoginData? this would help me knowing the payload I will send will work or not
```
.bodyValue("""{
  "urn": "fdc:gov.uk:2022:T5fYp6sYl3DdYNF0tDfZtF-c4ZKewWRLw8YGcy6oEj8",
   "otp": "123456",
   "email": "chrisy.clemence@gmail.com",
   "dob": "1982-10-24"
}"""

please

```

Reply:
@abdulrkalam is that ok?

Reply:
ok